exercises/wordy/wordy_spec.rb

```diff
@@ -0,0 +1,136 @@
+require_relative "./wordy"
+
+RSpec.describe WordProblem do
+  it "can add one to one" do
+    problem = WordProblem.new("What is 1 plus 1?")
+
+    expect(problem.answer).to eq(2)
+  end
+
+  it "can add two to fifty three" do
+    skip "fix this test second - delete the `skip` lines to enable individual tests"
```

Comment:
Glad I asked.

If it is idiomatic for RSpec to use pending, then we should use `pending`.  Exercism should not influence this, the tests, if we provide them, should be idiomatic and instructive.   The documentation states to use `skip` only due to the use of it in regards to Minitest.